schema.cc

```diff
@@ -638,6 +640,141 @@ std::ostream& operator<<(std::ostream& os, const schema& s) {
     return os;
 }
 
+std::ostream& map_as_cql_param(std::ostream& os, const std::map<sstring, sstring>& map, bool first = true) {
+    for (auto i: map) {
+        if (first) {
+            first = false;
+        } else {
+            os << ",";
+        }
+        os << "'" << i.first << "': '" << i.second << "'";
+    }
+
+    return os;
+}
+
+std::ostream& column_definition_as_cql_key(std::ostream& os, const column_definition & cd) {
+    os << cd.name_as_cql_string();
+    os << " " << cd.type->cql3_type_name();
```

Comment:
Please consider: What happens when UDT are used? Do we need to save the UDT definition in the CQL file as well? Does this cql3_type_name() method (I'm not familiar with it) work correctly also with UDTs and also with UDTs with quoted names (I believe this is possible, but didn't check...)
If UDTs don't work, we can leave it as a TODO, but we should throw an exception from describe() rather than silently doing the wrong thing.

Reply:
UDT and UDF are defined at the keyspace level. We assume that the keyspace is defined correctly before creating the table.

you are right that I should test it as well

Reply:
I keep asking and don't understand the response - is it fine that there is no maybe_quote on the cql3_type_name?